tests/rptest/tests/schema_registry_test.py

```diff
@@ -65,6 +65,7 @@ def create_topic_names(count):
 
 log_config = LoggingConfig('info',
                            logger_levels={
+                               'admin_api_server': 'trace',
```

Comment:
nit: This might be more noise than signal.

Reply:
Done in https://github.com/redpanda-data/redpanda/pull/8127/commits/f0417002c0a8d5772987cb92f736fc3543f94375